src/Formatter/JsonFormatter.php

```diff
@@ -20,10 +21,10 @@ public function format(array $record): string
             'lines' => [
                 [
                     'timestamp' => $record['datetime']->getTimestamp(),
-                    'line'      => $record['message'],
-                    'app'       => $record['channel'],
-                    'level'     => $record['level_name'],
-                    'meta'      => $record['context'],
+                    'line' => $record['message'],
```

Comment:
Really?

Reply:
I'll be simplifying this PR shortly by bringing in https://github.com/fusionspim/pint-coding-style/pull/1